### PR TITLE
Sameera/lesson item component

### PR DESCRIPTION
**Pull request by sameerav**
@sonia-y

### CODE REVIEW

File: app/assets/javascripts/components/AdminPanel/lessons_page.es6.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class LessonsPage extends React.Component {
```
Comment (sonia-y) [inline app/assets/javascripts/components/AdminPanel/lessons_page.es6.jsx]: Can you pull from master and put this file under components/admin? I reorganized the folders c:

File: app/assets/javascripts/components/AdminPanel/lessons_page.es6.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    return (
+      <LessonItem lesson={lesson} />
+    );
+  }
+
+  renderLessons() {
+    return this.props.lessons.map((lesson) => this.renderLesson(lesson));
+  }
+
+  render () {
```
Comment (sonia-y) [inline app/assets/javascripts/components/AdminPanel/lessons_page.es6.jsx]: Can you follow the format for the rest of the pages so far? So outside div should have classname page-wrapper, then the div with all the content should be content-wrapper. Also, you shouldn't have to wrap the adminheader inside another div.

File: app/assets/javascripts/components/AdminPanel/lessons_page.es6.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+          <AdminHeader />
+        </div>
+        <div className="page-wrapper">
+          {this.renderLessons()}
+        </div>
+      </div>
+    );
+  }
+}
+
```
Comment (sonia-y) [inline app/assets/javascripts/components/AdminPanel/lessons_page.es6.jsx]: delete this extra line!

File: app/assets/javascripts/components/lesson_item.es6.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+class LessonItem extends React.Component {
+
+  static get propTypes() {
+    return {
+      lesson: React.PropTypes.object
+    };
+  }
+
+  render () {
+    console.log(this.props.lesson)
```
Comment (sonia-y) [inline app/assets/javascripts/components/lesson_item.es6.jsx]: delete console.log

File: app/assets/javascripts/components/lesson_item.es6.jsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+class LessonItem extends React.Component {
+
+  static get propTypes() {
+    return {
+      lesson: React.PropTypes.object
+    };
```
Comment (sonia-y) [inline app/assets/javascripts/components/lesson_item.es6.jsx]: add trailing comma

File: app/assets/javascripts/components/lesson_item.es6.jsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+      <div className="lesson-card">
+        <div>Teacher id: {this.props.lesson.teacher_id}</div>
+        <div>Student id: {this.props.lesson.student_id}</div>
+        <div>Start time: {this.props.lesson.start_time}</div>
+        <div>End time: {this.props.lesson.end_time}</div>
+        <div>Price: {this.props.lesson.price}</div>
+      </div>
+    );
+  }
+}
```
Comment (sonia-y) [inline app/assets/javascripts/components/lesson_item.es6.jsx]: add line after

File: app/assets/stylesheets/components/lessons_item.scss [hunk 7]
```diff
@@ -0,0 +1,5 @@
+.lesson-card {
+	background-color: $grey20;
+	padding: 10px 20px 10px 20px;
+	margin: 10px 0px 10px 0px;
+}
```
Comment (sonia-y) [inline app/assets/stylesheets/components/lessons_item.scss]: add line after

File: app/assets/stylesheets/components/lessons_item.scss [hunk 8]
```diff
@@ -0,0 +1,5 @@
+.lesson-card {
+	background-color: $grey20;
+	padding: 10px 20px 10px 20px;
+	margin: 10px 0px 10px 0px;
```
Comment (sonia-y) [inline app/assets/stylesheets/components/lessons_item.scss]: can you shorten this to padding: 10px 20px and margin:10px 0px

File: app/views/admin/lessons.html.erb [hunk 9]
```diff
@@ -1,2 +1 @@
-<h1>Admin#lessons</h1>
-<p>Find me in app/views/admin/lessons.html.erb</p>
+<%= react_component('LessonsPage', {lessons: @lessons}) %>
```
Comment (sonia-y) [inline app/views/admin/lessons.html.erb]: sorry, super nitpicky but can you add spaces after/before bracket so { lessons: @lessons }

**Comment by sameerav** [comment]
@sonia-y fixed
